src/test/java/com/rultor/agents/req/DecryptTest.java

```diff
@@ -165,10 +165,8 @@ public void testHttpProxyHandling() throws IOException {
      */
     private XMLDocument createTestProfileXML() {
         return new XMLDocument(
-            Joiner.on(' ').join(
-                "<p><entry key='decrypt'><entry key='a.txt'>",
-                "a.txt.asc</entry></entry></p>"
-            )
+            // @checkstyle LineLength (1 line)
```

Comment:
@cvrebert let's keep the Joiner, but join on "nothing" (`''`)  and remove this checkstyle ignore...